addons/sale/views/sale_portal_templates.xml

```diff
@@ -127,7 +127,7 @@
                     <t t-set="classes" t-value="'col-lg-auto d-print-none'"/>
 
                     <t t-set="title">
-                        <h2 class="mb-0"><b t-field="sale_order.amount_total"/> </h2>
+                        <h2 class="mb-0"><b t-field="sale_order.amount_total" data-id="total_amount"/> </h2>
```

Comment:
Is there a benefit to use data-id instead of a class?

Reply:
Well, no, there is none, but identifying fields started that way, so I'm just continuing in that direction.
Plus, using CSS class not to apply style but for identifying data seems odd